portfolio/src/main/webapp/community.js

```diff
@@ -0,0 +1,40 @@
+function renderComments() {
+let quantity = document.getElementById("quantity").value;
+  fetch('/comments?quantity=' + quantity).then(response => response.json()).then((conversation) => {
+    console.log(conversation);
+
+    let commentSection = document.getElementById('submitted-comments');
+    while (commentSection.lastChild) {
+        commentSection.removeChild(commentSection.lastChild);
+    }
+
+    conversation.forEach((comment) => {
+        createComment(comment);
+    });
+  });
```

Comment:
You shouldn't need all the brackets and such here, I don't think. Try:

conversation.forEach(comment => createComment(comment));

Reply:
Made brackets necessary by adding a second line